ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -158,6 +163,24 @@ public class HelixBootstrapUpgradeUtil {
   private static final Logger logger = LoggerFactory.getLogger("Helix bootstrap tool");
   private static final String ALL = "all";
 
+  // Waged auto rebalancer default configs. TODO: Provide them as configurable inputs to tool.
+  static final String DISK_KEY = "DISK";
+  static final String RACK_KEY = "rack";
+  static final String HOST_KEY = "host";
+  static final String TOPOLOGY = "/" + RACK_KEY + "/" + HOST_KEY;
+  static final String FAULT_ZONE_TYPE = "rack";
+  static final int MAX_PARTITIONS_IN_TRANSITION = 50;
+  static final int DELAY_REBALANCE_TIME_IN_MS = 1800000;
+  static final int LESS_MOVEMENT = 2;
+  static final int EVENNESS = 3;
+  // the default of partition. each partition can store up to 24 * 16 = 384 GB data, and leave 2 GB here for margin
+  static final int DEFAULT_PARTITION_WEIGHT = 100;
```

Comment:
different clusters has different partition weight, in perf, it's 100, but in other cluster, it's 386.. should we nuke perf cluster and recreate it with 384 GB for partition? if not, we should set this value as an argument from cli.

Reply:
Sure. Took the arguments from cli